backend/src/database/migrations/20230705102914-index-sms-transactional-message-id-column.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+module.exports = {
+up: async (queryInterface, _) => {
```

Comment:
/nit alignment (you guys have no linting huh 😂)

Reply:
lol weird, we do, but somehow it's not being applied. thanks, will fix.